invenio_app_rdm/config.py

```diff
@@ -494,6 +498,27 @@ def files_rest_permission_factory(obj, action):
 # is solved
 CSRF_HEADER = "X-CSRFToken"
 
+# Invenio-Vocabularies
+# =============
+
+VOCABULARIES_DATASTREAM_READERS = {
+    **VOCABULARIES_DATASTREAM_READERS,
+    "orcid-http": OrcidHTTPReader,
```

Comment:
Minor thought here (to be shelved when we revisit contrib modules config injection), would it make sense to expose in each invenio-vocabularies contrib a config variable for `readers/transformers/writers`, so that one can e.g. easily get all of the latest config like so:

```python
from invenio_vocabularies.config import VOCABULARIES_DATASTREAM_TRANSFORMERS
import invenio_vocabularies.contrib.names.config as names_config

VOCABULARIES_DATASTREAM_TRANSFORMERS = {
    **VOCABULARIES_DATASTREAM_TRANSFORMERS,
    **names_config.VOCABULARIES_DATASTREAM_TRANSFORMERS,
}
```

Reply:
This actually makes a lot of sense, and I think is already possible... I'll do the changes